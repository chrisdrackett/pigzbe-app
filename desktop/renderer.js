import {AppRegistry} from 'react-native';
import App from './bundle.web.js';

AppRegistry.registerComponent('App', () => App);
AppRegistry.runApplication('App', {rootTag: document.getElementById('react-root')});
