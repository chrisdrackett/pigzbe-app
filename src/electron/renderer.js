import {AppRegistry} from 'react-native';
// import App from '../../App';
import App from '../../dist/bundle.web.js';

AppRegistry.registerComponent('App', () => App);
AppRegistry.runApplication('App', {rootTag: document.getElementById('react-root')});
