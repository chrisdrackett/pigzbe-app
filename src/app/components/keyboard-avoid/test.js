import React from 'react';
import renderer from 'react-test-renderer';
import KeyboardAvoid from './';

const props = {};

describe('Keyboard Avoid', () => {
    test('renders correctly', () => {
        const tree = renderer.create((
            <KeyboardAvoid {...props}/>
        )).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
