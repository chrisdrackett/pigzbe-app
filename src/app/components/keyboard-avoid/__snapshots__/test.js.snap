// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Keyboard Avoid renders correctly 1`] = `
<View
  contentContainerStyle={undefined}
  enabled={true}
  keyboardVerticalOffset={0}
  onLayout={[Function]}
  style={
    Object {
      "flex": 1,
    }
  }
>
  <View
    style={
      Array [
        undefined,
        Object {
          "bottom": 0,
        },
      ]
    }
  />
</View>
`;
